opentmi_client/opentmi_client.py

```diff
@@ -210,6 +217,8 @@ def __get_campaigns(self):
         return self.__get_JSON(url)
 
     def __get_suite(self, suite, options=''):
+        suite = suite if isinstance(suite, basestring) else str(suite)
```

Comment:
This basestring-check might not be very robust on both python 2 and python 3. Would it be better to just make sure that __get_suite is not called if suite is not a string? This is an internal helper, not a public api so we should have control over where it is called. The other option would be to do some compatibility hacks through six module (I think it has a six.stringtypes or somesuch which at least tries to be an ok solution for both python 2 and 3).